.circleci/config.yml

```
@@ -30,6 +30,8 @@ job_default: &job_defaults
         - /\d+\.\d+\.x/
 
 push_images: &push_images
+  environment:
+    DOCKER_IMAGES_TO_PUSH: "komodo-server syndesis-meta syndesis-operator syndesis-s2i syndesis-server syndesis-ui syndesis-ui-legacy syndesis-upgrade"
```

```suggestion
    DOCKER_IMAGES_TO_PUSH: "syndesis-meta syndesis-operator syndesis-s2i syndesis-server syndesis-ui syndesis-ui-legacy syndesis-upgrade"
```

I don't think `komodo-server` is part of this CircleCI workflow